metricbeat/module/http/server/data.go

```diff
@@ -59,21 +59,21 @@ func (m *metricProcessor) RemovePath(path PathConfig) {
 	m.Unlock()
 }
 
-func (p *metricProcessor) Process(event server.Event) (common.MapStr, error) {
```

Comment:
Why is this change needed?

Reply:
will not be an issue with `NewHttpServerWithHandler`